site/en/docs/crux/release-notes/index.md

```diff
@@ -43,6 +43,21 @@ The CrUX dataset on BigQuery is generally updated on the second Tuesday of every
 
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.
 
+## 202310
+
+[Announcement](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/??)
+
+Publication date
+ : November 14, 2023
+
+What's new
+ : - We're again seeing continual improvements across most of the metrics likely due to the final rollout of the [front end improvements to LCP started in September](#202308) which particularly affected LCP (up a further 1.5% this month) and CLS (up 1.6%)
```

Comment:
Is it more accurate to say that the improvements themselves started in August?

```suggestion
 : - We're again seeing continual improvements across most of the metrics likely due to the final rollout of the [front end improvements to LCP started in August](#202308) which particularly affected LCP (up a further 1.5% this month) and CLS (up 1.6%)
```

Reply:
Yeah. It was an experiment so taken a while to get up to 100% (hence why still seeing impact 2 releases later). It was at 50% for part of August.